freqtrade/configuration/cli_options.py

```diff
@@ -198,6 +198,12 @@ def __init__(self, *args, **kwargs):
         action='store_false',
         default=True,
     ),
+    "print_json": Arg(
+        '--print-json',
```

Comment:
i am not sure about this argument naming

wouldn't `--print-result-format json` be more flexible (so we can add `csv` / `strategycode` / whatever other format) later?

Reply:
I was also thinking of this kind of option, but my approach is that if (when) `--print-result-format=json/csv/python/whatever` is introduced, the `--print-result-json`, `--print-result-csv`, etc. options should be left. The user will be able to choose more convenient approach to specify format. In this case in the description of `--print-result-json` it may be stated that it's equivalent to `--print-result-format=json`. This approach (`--format=val` and corresponding one-argument shortcut option) can frequently be found in utilities with wide set of options.

For now I would like to leave this option, I can rename it to `--print-result-json`.

Reply:
nah, what you got is fine :+1: 

makes sense to eventually extend to "result_format"...